manifests/params.pp

```diff
@@ -22,13 +22,21 @@
   case $facts['os']['family'] {
     'Debian': {
       if $facts['os']['release']['major'] =~ /^(9|10)$/ {
+        $debian_nodejs_dev_package_name = $facts['os']['release']['major'] ? {
+          '9'     => 'nodejs-dev',
```

Comment:
can you add an accetpance test to verify the new settings are correct?

Reply:
@bastelfreak I added acceptance tests. I'm not sure if the "uninstall" step is the proper way to do this, but it works.